xla/service/gpu/fusions/cudnn_test.cc

```diff
@@ -14,6 +14,9 @@ limitations under the License.
 ==============================================================================*/
 
 #include <gtest/gtest.h>
+#include "absl/strings/str_replace.h"
```

Comment:
Need the corresponding BUILD file update for these headers I think

Reply:
Done.